File: Topic Based Problems/01 Fundamentals/Beginner/19 Pounds to Kilograms/Java/PoundstoKilograms.java
/**
 * @file PoundstoKilograms.java
 * @author SharifdotG
 * @brief Pounds to Kilograms in Java
 * @version 1.0
 * @date 2023-09-24
 * 
 */

import java.util.Scanner;

public class PoundstoKilograms {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double pounds;

        System.out.print("Enter the weight in Pounds: ");
        pounds = scanner.nextDouble();

        System.out.printf("Weight in Kilograms = %.2f\n", pounds * 0.453592);

        scanner.close();
    }
}

File: Topic Based Problems/01 Fundamentals/01 Beginner/19 Pounds to Kilograms/C/PoundstoKilograms.c
/**
 * @file PoundstoKilograms.c
 * @author SharifdotG (sharifmdyousuf007@gmail.com)
 * @brief Pounds to Kilograms in C
 * @version 1.0
 * @date 2023-09-24
 * 
 * @copyright Copyright (c) 2023
 * 
 */

#include <stdio.h>

int main() {
    double pounds;

    printf("Enter the weight in Pounds: ");
    scanf("%lf", &pounds);

    printf("Weight in Kilograms = %.2lf\n", pounds * 0.453592);

    return 0;
}
